api/app/channels/plugins/matrix/channel.py

```
@@ -380,13 +382,18 @@ def get_delivery_target(self, metadata: dict[str, Any]) -> str:
         return metadata.get("room_id", "")
 
     def format_escalation_message(
-        self, username: str, escalation_id: int, support_handle: str
+        self,
+        username: str,
```

Ruff (0.15.2)

[warning] Unused method argument: `username`

(ARG002)